### ISSUE TITLE
Create documentation

### ISSUE THREAD

**Issue by robdmoore** [TestStack/TestStack.Dossier#28]
We will have a readme.io site soon to add to cc @mwhelan

**Comment by mwhelan** [comment]
Awesome, look forward to helping with that!

**Comment by robdmoore** [comment]
Most of the initial hard work will be done by a blog post I'm about to hit the go button on

**Comment by mwhelan** [comment]
Even better!! :-)

**Comment by mwhelan** [comment]
Would be good to re-post that on the TestStack blog, which will be live shortly too once we finalise readme.io stuff....

**Comment by robdmoore** [comment]
Yep, happy to, although we need to work out how that will work - Google doesn't like duplicated content. I guess we can do a summary of the post and link off?

Post is live now: http://robdmoore.id.au/blog/2015/04/22/announcing-teststack-dossier-library/
